lib/rucio/daemons/transmogrifier/transmogrifier.py

```diff
@@ -589,20 +592,26 @@ def run_once(heartbeat_handler: "HeartbeatHandler", bulk: int, **_kwargs) -> boo
                     nb_rule = 0
                     #  Try to create the rule
                     logger(logging.DEBUG, 'selected_rses : %s' % selected_rses)
-                    try:
-                        for rse in selected_rses:
-                            if isinstance(selected_rses, dict):
-                                #  selected_rses is a dictionary only when split_rule is True or for chained subscriptions
-                                source_replica_expression = selected_rses[rse].get(
-                                    "source_replica_expression",
-                                    None,
-                                )
-                                weight = selected_rses[rse].get("weight", None)
-                            logger(
-                                logging.INFO,
-                                "Will insert one rule for %s:%s on %s"
-                                % (did["scope"], did["name"], rse),
+                    for rse in selected_rses:
+                        if isinstance(selected_rses, dict):
+                            #  selected_rses is a dictionary only when split_rule is True or for chained subscriptions
+                            source_replica_expression = selected_rses[rse].get(
+                                "source_replica_expression",
+                                None,
                             )
+                            weight = selected_rses[rse].get("weight", None)
+                        logger(
+                            logging.INFO,
+                            "Will insert one rule for %s:%s on %s"
+                            % (did["scope"], did["name"], rse),
+                        )
```

Comment:
Please pass the variables as arguments to the `logger()` call instead of using string interpolation (see https://docs.astral.sh/ruff/rules/logging-percent-format/ for the justification).